lib/wallet/src/Cardano/Wallet.hs

```diff
@@ -3854,6 +3892,13 @@ data PoolRetirementEpochInfo = PoolRetirementEpochInfo
     }
     deriving (Eq, Generic, Show)
 
+throwOnErr :: (MonadIO m, Exception e) => Either e a -> m a
```

Comment:
I wonder if those things should not be somewhere else to be reused by other modules and packages... Possible here https://github.com/cardano-foundation/cardano-wallet/blob/master/lib/wallet/src/UnliftIO/Compat.hs or somewhere close to it. Maybe this is proper place as it is not needed elsewhere

Reply:
I agree it would be nice to co-locate these helpers.

But perhaps somewhere else other than `UnliftIO`, as that is specific to the `unliftio` way of doing things.

Thoughts?